src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -651,14 +710,26 @@ private int constructMergedTable(Table table, List<FeedToMerge> feedsToMerge,
                                     int startDateIndex =
                                         getFieldIndex(fieldsFoundInZip, "start_date");
                                     LocalDate startDate = LocalDate
-                                        .parse(csvReader.get(startDateIndex),
-                                            GTFS_DATE_FORMATTER);
+                                        .parse(csvReader.get(startDateIndex), GTFS_DATE_FORMATTER);
                                     if (feedIndex == 0) {
                                         // For the future feed, check if the calendar's start date is earlier than the
                                         // previous earliest value and update if so.
                                         if (futureFirstCalendarStartDate.isAfter(startDate)) {
                                             futureFirstCalendarStartDate = startDate;
                                         }
+                                        // FIXME: Move this below so that a cloned service doesn't get prematurely
+                                        //  modified? (do we want the cloned record to have the original values?)
+                                        if (index == startDateIndex) {
+                                            if (EXTEND_FUTURE == mergeStrategy ||
+                                                (CHECK_STOP_TIMES == mergeStrategy &&
+                                                    // TODO: Need to ensure serviceIds are being extended.
```

Comment:
Can this comment be elaborated on a little further? Perhaps reference an issue if needed?

Reply:
This was outdated (lingering from when the behavior wasn't working as expected yet).